scripts/auto-rebase/rebase_job_entrypoint.sh

```diff
@@ -23,17 +23,18 @@ release_amd64="$(oc get configmap/release-release-images-latest -o yaml \
 release_arm64="$(oc get configmap/release-release-images-arm64-latest -o yaml \
     | yq '.data."release-images-arm64-latest.yaml"' \
     | jq -r '.metadata.name')"
-# LVMS is not tracked in the OCP release image.  Instead, rely on the latest X.Y stream as the release image.
-#  LVMS also does not cut nightly releases where ocp-release does.  This means that latest ocp-releases' y-stream
-#  can increment independently from LVMS, and will usually be 1 y-stream ahead of LVMS in-between OCP releases.
-#  For example, ocp-release at 4.13 will more often than not correspond to 4.12 LVMS, until the official 4.13 release when
-#  both components will be 4.13.
-release_lvms="v4.12"
 
 pullspec_release_amd64="registry.ci.openshift.org/ocp/release:${release_amd64}"
 pullspec_release_arm64="registry.ci.openshift.org/ocp-arm64/release-arm64:${release_arm64}"
-# Since LVMS is not part of the release payload, it is not kept in CI. Use the latest z-stream that coincides with the release payload's X.Y version
-pullspec_release_lvms="registry.access.redhat.com/lvms4/lvms-operator-bundle:${release_lvms}"
+
+branch_name="$(git branch --show-current)"
```

Comment:
Does the build process check out a branch by reference, or does it check out a SHA directly?

Reply:
It's by reference. I ran a [rebase pj-rehearse on r-4.12](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36455/rehearse-36455-periodic-ci-openshift-microshift-release-4.12-periodics-rebase-on-nightlies/1626161071553253376/artifacts/rebase-on-nightlies/openshift-microshift-rebase/build-log.txt) with just `git status` and `git branch`, and this was the output:
```
On branch release-4.12
nothing to commit, working tree clean
* release-4.12
```

Reply:
OK, that should work then.